### DISCUSSION TITLE
Maybe add the ability to comment on projects?

### DISCUSSION THREAD

**Discussion by las-r** [modrinth/.github#272] [Feature requests]
I want the ability to comment on projects. Obviously this will require some pretty decently heavy moderation, but if it's possible to find the workers, then comments would genuinely be amazing for feedback and just overall a more integrated community.

## User Side
Below each project, all the way at the bottom will be the comments section, or possibly the section could be off to the side. They can comment things, along with being able to reply to and like / vote on other comments. For the like / vote system, I feel like either just having a like counter, without any option to dislike, would be nice. An upvote / downvote similar to Reddit would also work nicely. This, along with a report system, would be quite nice.

## Creator Side
Creators will be able to comment as well, with their comments having a Creator or Project Member tag. They can also pin other peoples comments to the side. The project will also have options to limit comments to project members, or just disable comments altogether. Other than these, this side will be nearly identical to the user side.

## Moderator Side
Moderators will be able to directly delete comments. Their comments will have a Moderator tag.

Obviously, this is just an idea that needs more work upon. But, for now, this is a solid concept that will bring the Modrinth community more forward and help out creators.

**Comment by AnOpenSauceDev** [comment]
This is on the [roadmap](https://support.modrinth.com/en/articles/8800954-modrinth-roadmap), but I haven't seen much happening about it.
